addons/web/static/src/js/views/list/list_editable_renderer.js

```diff
@@ -383,9 +383,11 @@ ListRenderer.include({
 
         // Toggle selected class here so that style is applied at the end
         $row.toggleClass('o_selected_row', editMode);
-        $row.find('.o_list_record_selector input').prop('disabled', !record.res_id)
+        $row.find('.o_list_record_selector input').prop('disabled', !record.res_id);
 
-        return $.when.apply($, defs);
+        return $.when.apply($, defs).then(function () {
+            core.bus.trigger('DOM_updated');
```

Comment:
add comment here (the same as in BasicRenderer)

Reply:
Same as in BasicController you mean?

// necessary to allow all sub widgets to use their dimensions in
// related activities, such as autoresize on fieldtexts

Shouldn't that be documented only in one place instead of being duplicated?